assets/app/models/Site.js

```diff
@@ -0,0 +1,23 @@
+var Backbone = require('backbone');
+
+var SiteModel = Backbone.Model.extend({
+  defaults: {
+    'builds': [],
+    'repository': 'Repo goes here',
+    'engine': 'jekyll',
+    'branch': 'master'
+  },
+  urlRoot: '/v0/site'
```

Comment:
Is this redundant with the `collection.url`? Should we pull out the `/v0` into a global `api.version` var, or somehow automatically prefix it to the request? Or we could follow GitHub's lead and version the API with a header instead of in the URL.